assets/src/dashboard/app/reducer/stories.js

```diff
@@ -58,6 +60,32 @@ function storyReducer(state, action) {
         },
       };
 
+    case ACTION_TYPES.TRASH_STORY:
+      return {
+        ...state,
+        storiesOrderById: state.storiesOrderById.filter(
+          (id) => id !== action.payload.id
+        ),
+        totalStories: state.totalStories - 1,
+        stories: Object.keys(state.stories).reduce((memo, storyId) => {
+          if (parseInt(storyId) !== action.payload.id) {
+            memo[storyId] = state.stories[storyId];
+          }
+          return memo;
+        }, {}),
+      };
+
+    case ACTION_TYPES.DUPLICATE_STORY:
+      return {
+        ...state,
+        storiesOrderById: [...state.storiesOrderById, action.payload.id],
```

Comment:
Does order matter here? Should the duplicated story be forced to the top of the order?

Reply:
I agree with this question, I feel if I were to duplicate something I'd want it to appear somewhere in the list where I could see it... I almost think it could just appear next to the story it duplicated even if that breaks the overall sort 🤔 But maybe that's UX question for Sam and can be implemented in a later PR.

Reply:
I will update it so it shows up first. The only way to get it in sorted order is to call the entire re-fetch command which could actually end up hiding it if the sort order places it on another page.